resources/migrations/001_update_migrations.yaml

```diff
@@ -10087,6 +10087,14 @@ databaseChangeLog:
                   constraints:
                     nullable: true
 
+  - changeSet:
+      id: v52.2024-11-15T10:00:00
+      author: adam-james
+      comment: Add graph.max_categories_enabled to bar chart viz-settings, defaulting to false
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.CreateGraphMaxCategoriesEnabledVizSetting"
```

Comment:
Is there a way to add this without needing a database migration? These sorts of migrations are slow since they need to do full table scans. 

If they key is not present, can't you just treat it as `false`?